proprietary/design-tokens/src/imported/nl/utrecht-radio-button.json

```diff
@@ -0,0 +1,198 @@
+{
+  "utrecht": {
+    "radio-button": {
+      "border-width": {
+        "value": "1.5px",
```

Comment:
Moet dit echt niet meeschalen met de `font-size`?

Reply:
De border width meeschalen met de font-size?

Reply:
Door `rem` te gebruiken schaalt het mee met de `font-size`. Met `px` zet je de maat vast, ongeacht de instellingen van de gebruiker. 

`16px` is de browser standaard maar sommige gebruikers veranderen die waarde in hun browser naar wat zij fijn vinden. 

Er is hier geen goed of fout, alleen bewust en onbewust.

Reply:
Aaah dat is waar. Tot nu toe doen we voor borders px om de reden dat die niet onnodig dik worden. Maar in dit geval zou je inderdaad kunnen wensen om er rem van te maken. Ik ga even navragen wat andere hiervan vinden.